src/components/Fetch.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+
+const FetchedData = () => {
+  const [loading, setLoading] = useState(false);
```

Comment:
[ Optional Change ]

- [ ] It is a good naming convention to name your file or function as meaningful as it does. For this case, this file is meant for fetching Quotes, hence, you could have named the file Quotes as well as the function could have been named the same as the project requirement stated. 👍🏽
- [ ] Let's always try to follow project guidelines as we're being trained for real job places 👍🏽